content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -637,7 +637,8 @@ The access modes are:
 : the volume can be mounted as read-write by many nodes.
 
  `ReadWriteOncePod`
-: the volume can be mounted as read-write by a single Pod. Use ReadWriteOncePod
+: {{< feature-state for_k8s_version="v1.27" state="beta" >}}
```

Comment:
Not sure if this `feature-state` shortcode works properly under a definition...
The preview cannot be checked...

Reply:
It works -- the preview can be found at:
https://deploy-preview-39843--kubernetes-io-main-staging.netlify.app/docs/concepts/storage/persistent-volumes/

![Screenshot 2023-03-14 at 1 48 50 PM](https://user-images.githubusercontent.com/3466879/225439350-53ad8928-b762-41d1-b6a2-b15874312db3.png)